### DISCUSSION TITLE
Remove printed METAR/TEXT piece of paper at the end of a flight

### DISCUSSION THREAD

**Discussion by pdeswardt** [flybywiresim/aircraft#6309] [Ideas]
**Scenario:** 

_You complete a flight, and arrive at the destination airport gate and shut your plane down._ 

**Suggestion:**
Once the engines are shut down, any METAR or other printed TEXT that you moved to the pedestal ( white piece of paper ) should be removed, as it's not longer needed for the completed flight. A new printout will be printed for the next flight, assuming that you will either do a turnaround flight, or another leg as part of a longer trip.

I know all new prints will just lay over the top of existing printouts, but maybe there can be an option to remove old prints at the pilot's discretion at least ?

Thanks for this consideration.

**Comment by tracernz** [comment]
You can remove them yourself, by clicking on the bottom edge.

**Comment by pdeswardt** [comment]
Ahhh, thanks. Never knew that!

Sorry for wasting everyone's time.

Thanks!

**Reply by tracernz** [reply]
No problem at all. :)

**Reply by CptBayer** [reply]
after removing it I wanna see the piece of paper in the dust bin  :-p ....

**Comment by andrejetcat** [comment]
It would be nice, if you can print out e.g. your Callsign (or kind like self - drawing) and pin it up on the glaresheeld as we pilots do it IRL :)
